aspnetcore/tutorials/Microsoft.dotnet-openapi.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Develop ASP.NET Core apps using OpenAPI
+author: ryanbrandenburg
+description: This tutorial demonstrates how to use the 'Microsoft.dotnet-openapi' tool to add references to OpenAPI files 
```

Comment:
Tutorial? Reads more like a tool description and I suggest placing it somewhere else.

Reply:
Where would you suggest? I put it alongside dotnet-watch because that's another tool, but it's unclear where would be a better fit.

Reply:
@Rick-Anderson ??